src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -42,7 +40,11 @@ public class TelegramBotClient : ITelegramBotClient
 
         private static readonly JsonSerializerSettings SerializerSettings = new JsonSerializerSettings
         {
-            NullValueHandling = NullValueHandling.Ignore
+            NullValueHandling = NullValueHandling.Ignore,
+            ContractResolver = new DefaultContractResolver
```

Comment:
I know request serialization are only in library but I'd prefer to create a set of reusable request classes that could potentially be used outside of Bot API and don't rely on specific serialization settings to be provided.

Reply:
Could you expand on it a little more? I don't understand what you mean by "outside of Bot API".

Reply:
Some of our request classes could be used in other tg apps such as a tg client.

Also, users now can log/serialize requests before sending them so it is necessary to have all required serialization attributes on classes. No need to require users to pass serialixation settings.

Reply:
Yeah, valid point about logging requests.

Reply:
Though not all requests can be serialized to json. E.g. requests with files. You also commented on `SetWebhookRequest.cs` that json attributes can be ommited because it is always use  `multipart/form-data`. How can you log such requests?

Reply:
We could add an extension method for that.
```c#
// Generate HTTP Content and then stringify it
string logMe = reqeust.ToPayloadString()
```
this method would also be used in `SendingRequest` and `ResponseReceived` events